src/API.jl

```diff
@@ -894,15 +894,33 @@ end
 
 precompile() = precompile(Context())
 function precompile(ctx::Context)
+    
+    is_stdlib_from_name(name::String) = name in values(stdlibs())
```

Comment:
This needs to go away. Check the uuid.

Reply:
Fixed. Didn't quite get how to do it initially